modular_zubbers/modules/ballstics/wt_autorifle.dm

```diff
@@ -0,0 +1,192 @@
+// WT-551 Autorifle
+
+/obj/item/gun/ballistic/automatic/wt550/security
+	name = "\improper WT-551 Autorifle"
+	desc = "A heavier, bulkier automatic variant of the WT-550, and now with 99% less discombobulation! It's back, baby. Uses 4.6x30mm rounds. Recommended to hold with two hands."
+	icon = 'modular_zubbers/icons/obj/weapons/guns/wt551.dmi'
+	fire_sound = 'modular_zubbers/sound/weapons/gun/wt551/shot.ogg'
+	w_class = WEIGHT_CLASS_BULKY
+	fire_delay = 3
+	//18 damage per 0.3 seconds = 60 DPS
+	//Reference: Laser Gun 22 damage per 0.4 seconds = 55DPS
+
+/obj/item/gun/ballistic/automatic/wt550/security/flathead //What you get in the armory.
+	spawn_magazine_type = /obj/item/ammo_box/magazine/wt550m9/flathead
+
+/obj/item/gun/ballistic/automatic/wt550/security/rubber //What you get from cargo.
+	spawn_magazine_type = /obj/item/ammo_box/magazine/wt550m9/rubber
+
+//PROJECTILES
+/obj/projectile/bullet/c46x30mm/flathead
+	name = "4.6x30mm flathead bullet"
+
+	damage = 18
+	stamina = 5 //Knocks the wind out of you.
+
+	wound_bonus = CANT_WOUND
+	bare_wound_bonus = CANT_WOUND
+
+	weak_against_armour = TRUE
+	sharpness = NONE
+	shrapnel_type = null
+	embedding = null
+
+/obj/projectile/bullet/c46x30mm/rubber
+	name = "4.6x30mm rubber-tipped bullet"
+	damage = 5
+	stamina = 15
```

Comment:
```suggestion
	stamina = 16
```
could have it at 16 instead so it's a  7.5 hit till you get dropped from stamina damages

Reply:
I think 15 is fine for now unless people think it's too weak.